### DISCUSSION TITLE
consistent crash when using this submarine in outpost deathmatch (crash report attached)

### DISCUSSION THREAD

**Discussion by girlyguppy** [FakeFishGames/Barotrauma#15047] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

[beacon vs beacon.zip](https://github.com/user-attachments/files/17609072/beacon.vs.beacon.zip)
when using this submarine in outpost deathmatch, it loads in fine, and then crashes after a few seconds.

crash reports (all similar): 
[servercrashreport (5).log](https://github.com/user-attachments/files/17609077/servercrashreport.5.log)
[servercrashreport (8).log](https://github.com/user-attachments/files/17609078/servercrashreport.8.log)
[servercrashreport (7).log](https://github.com/user-attachments/files/17609079/servercrashreport.7.log)
[servercrashreport (6).log](https://github.com/user-attachments/files/17609080/servercrashreport.6.log)

### Reproduction steps

1. use the provided submarine
2. enter outpost deathmatch
3. witness crash after loading in

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/342e217046db64ee4344852704fb1c6869e680de (commit in our private repo), fix coming up in the next unstable build.
